app/models/reservation.rb

```diff
@@ -0,0 +1,7 @@
+class Reservation < ApplicationRecord
+  belongs_to :seance
+  belongs_to :user
+
+  validates :status, presence: true
+  enum :status, %i[created confirmed canceled]
```

Comment:
use explicit enums where possible
it makes it more readable for other developers
```suggestion
  enum status: { created: 0, confirmed: 1, canceled: 2 }
```

Reply:
Changed both enums, I'm gonna use explicit enums from now on :D.